26-register_registration/resource_types/artistic_result/creator_selects_resource_type_artistic_result_and_subtype_literary_arts.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Creator selects Resource type Artistic Result and subtype Literary Arts
+
+  Scenario: Creator navigates to the Resource Type tab and selects Resource subtype "Literary Arts"
+    Given Creator navigates to the Resource Type tab and selects Resource type "Artistic Result"
+    When they select Resource Subtype "Literary Arts"
+    Then they see fields:
+      | More information |
+    And they see field for Type Work with options:
+      | Novel        |
+      | Poetry       |
+      | Novella      |
+      | ShortFiction |
+      | Essay        |
+      | Translation  |
+      | Retelling    |
+      | Play         |
+      | Other        |
+    And they can add Announcements of type:
+      | Monograph                |
+      | Web Publication          |
+      | Performance              |
+      | Audio/Visual Publication |
+    And they see that each Announcement has a Remove Button
```

Comment:
Ikke mulig å se Remove Button eller Edit Button før du har lagt til en Announcement

Reply:
Legger til edit/remove for alle scenarioene under i stedet
```
And they can edit the the <TYPE>
And they can remove the the <TYPE>
```